dagstore.go

```diff
@@ -519,6 +519,17 @@ func (d *DAGStore) GetShardInfo(k shard.Key) (ShardInfo, error) {
 	return info, nil
 }
 
+func (d *DAGStore) Stats() map[ShardState]int {
+	d.lk.RLock()
+	defer d.lk.RUnlock()
+
+	stats := make(map[ShardState]int)
```

Comment:
Prealloc

Reply:
What do you think should be the capacity?

Reply:
It is grouping values by ShardState, there will be very few elements inside this map